### PR TITLE
[zh-cn]: create translation of XSLT Glossary

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/glossary/xslt/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: XSLT
+slug: Glossary/XSLT
+l10n:
+  sourceCommit: 5fea7c9593f5e4b4ef13ec65064acf1eabf01e4e
+---
+
+{{GlossarySidebar}}
+
+_可扩展样式表转换语言_ (**XSLT**) 是一种样式转换标记语言，用于将 {{Glossary("XML", "可扩展标记语言")}} 文档转换为其他 XML 文档、{{Glossary("HTML", "超文本标记语言")}}、{{Glossary("PDF", "可移植文档格式")}}、纯文本等。
```
Comment (jasonren0403) [inline files/zh-cn/glossary/xslt/index.md:10]:
```suggestion
_可扩展样式表转换语言_（**XSLT**）是一种样式转换标记语言，用于将{{Glossary("XML", "可扩展标记语言")}}文档转换为其他 XML 文档、{{Glossary("HTML", "超文本标记语言")}}、{{Glossary("PDF", "可移植文档格式")}}、纯文本等。
```

File: files/zh-cn/glossary/xslt/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+XSLT 处理器是一种接受 XML 输入或任何可转换为 XQuery 和 XPath 数据模型的格式的处理器。XSLT 处理器根据 XML 文档和 XSLT 样式表生成新的文档，在此过程中不会对原始文件进行任何更改。
+
+## 参见
+
+- 维基百科上的 [XSLT](https://en.wikipedia.org/wiki/XSLT)
```
Comment (jasonren0403) [inline files/zh-cn/glossary/xslt/index.md:16]: 尽可能使用本地化wiki源

```suggestion
- 维基百科上的 [XSLT](https://zh.wikipedia.org/wiki/XSLT)
```

File: files/zh-cn/glossary/xslt/index.md (hunk 1 above)
Comment (yin1999) [inline files/zh-cn/glossary/xslt/index.md:10]: We don’t need to translate these common proper nouns

```suggestion
_可扩展样式表转换语言_（**XSLT**）是一种声明式语言，用于将 {{Glossary("XML")}} 文档转换为其他 XML 文档、{{Glossary("HTML")}}、{{Glossary("PDF")}}、纯文本等。
```

File: files/zh-cn/glossary/xslt/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: XSLT
+slug: Glossary/XSLT
+l10n:
+  sourceCommit: 5fea7c9593f5e4b4ef13ec65064acf1eabf01e4e
```
Comment (jasonren0403) [inline files/zh-cn/glossary/xslt/index.md:5]:
```suggestion
  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
```

**Review by jasonren0403 — CHANGES_REQUESTED** [review]
一些修改建议

File: files/zh-cn/glossary/xslt/index.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+{{GlossarySidebar}}
+
+_可扩展样式表转换语言_ (**XSLT**) 是一种样式转换标记语言，用于将 {{Glossary("XML", "可扩展标记语言")}} 文档转换为其他 XML 文档、{{Glossary("HTML", "超文本标记语言")}}、{{Glossary("PDF", "可移植文档格式")}}、纯文本等。
+
+XSLT 处理器是一种接受 XML 输入或任何可转换为 XQuery 和 XPath 数据模型的格式的处理器。XSLT 处理器根据 XML 文档和 XSLT 样式表生成新的文档，在此过程中不会对原始文件进行任何更改。
```
Comment (yin1999) [inline files/zh-cn/glossary/xslt/index.md:12]:
```suggestion
XSLT 有它自己的可接受 XML 输入或任何可转换为 XQuery 和 XPath 数据模型的格式的处理器。XSLT 处理器根据 XML 文档和 XSLT 样式表生成新的文档，在此过程中不会对原始文件进行任何更改。
```

File: files/zh-cn/glossary/xslt/index.md (hunk 2 above)
Comment (yin1999) [inline files/zh-cn/glossary/xslt/index.md:16]:
```suggestion
- 维基百科上的 [XSLT](https://zh.wikipedia.org/wiki/XSLT)
```

**Review by yin1999 — COMMENTED** [review]
some suggestions:

**Review by yin1999 — APPROVED** [review]
left my +1 here

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**